app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def index
+    @user = User.all
+  end
+end
```

Comment:
- Good job done so far, but according to the project requirements, your controller should have a `show` action and a corresponding view file. Please fix this 🙏. Please note that it is okay for your controller to be empty 🙏